hypervisor/arch/x86/ioapic.c

```diff
@@ -311,6 +311,16 @@ irq_gsi_mask_unmask(uint32_t irq, bool mask)
 		irq, pin, rte.full);
 }
 
+void gsi_mask_irq(uint32_t irq)
+{
+	irq_gsi_mask_unmask(irq, true);
+}
+
+void gsi_unmask_irq(uint32_t irq)
+{
+	irq_gsi_mask_unmask(irq, false);
+}
+
```

Comment:
Do we want to keep these 2 APIs, or simply replace with irq_gsi_mask_unmask in caller side?
@lifeix @junjiemao1

Reply:
I prefer to keep them. To me, they're more readable. (Since they're the external APIs)